libraries/botbuilder-core/src/skypeMentionNormalizeMiddleware.ts

```diff
@@ -5,7 +5,9 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License.
  */
-import { Activity, Middleware, TurnContext } from 'botbuilder-core';
```

Comment:
Not sure why this change is here...

Reply:
This is necessary because it was telling a class inside of botbuilder-core to look inside the botbuilder-core's node_modules/ for **botbuilder-core**, which makes no logical sense.

It is part of the reason that we found the bug regarding the typescript 3.7. 

By looking for botbuilder-core, it tried to resolve this dependency through the normal Node.js fashion, and traversed the path upward where it found `botbuilder-core@4.6.1` installed at the root node_modules/.

Reply:
I also had to fix this in my rush branch. This is a good fix.